bin/lutris

```diff
@@ -47,18 +47,22 @@ try:
         gettext.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
         locale.textdomain("lutris")
         gettext.textdomain("lutris")
-    except:
+    except Exception as ex:
         sys.stderr.write(
             "Couldn't bind gettext domain, translations won't work.\n"
-            "LOCALE_DIR: %s\n" % optional_settings.LOCALE_DIR
+            "LOCALE_DIR: %s\nError: %s\n" % (optional_settings.LOCALE_DIR,ex)
         )
 except ImportError:
     pass
 
 if "WEBKIT_DISABLE_DMABUF_RENDERER" not in os.environ:
     os.environ["WEBKIT_DISABLE_DMABUF_RENDERER"] = "1"
 
-from lutris.gui.application import Application  # pylint: disable=no-name-in-module
+try:
+    from lutris.gui.application import Application  # pylint: disable=no-name-in-module
+except ImportError as ex:
+    sys.stderr.write("Error importing Lutris application module: %s\n" % ex)
+    sys.exit(1)
```

Comment:
Why do we want to catch this exception? We're going to exit if this fails in any case. What does this handler add?

Reply:
Here I was just trying to make the import error explicit, but it's true the program will exit anyways if you want I can revert it back.